CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ---------------------
 
 ### Added
-- Support for Simulation Mode on Windows.
+- Support `transition_using_threads` EDL attribute for ecalls in oeedger8r.
+  OE SDK now supports both switchless OCALLs and ECALLs.
```

Comment:
keep "Support for Simulation Mode on Windows."

Reply:
Looks like this still needs to be fixed

Reply:
Good catch. Readded. The simulation mode line item had somehow appeared under v0.8.1 tag. I have moved it to the right place under Unreleased.